lib/Backend/SwitchIRBuilder.cpp

```diff
@@ -163,7 +163,7 @@ SwitchIRBuilder::BeginSwitch()
 void
 SwitchIRBuilder::EndSwitch(uint32 offset, uint32 targetOffset)
 {
-    FlushCases(targetOffset);
+    FlushCases(offset);
     AssertMsg(m_caseNodes->Count() == 0, "Not all switch case nodes built by end of switch");
```

Comment:
What is offset/targetOffset here ? (I don't know this code)

Reply:
`offset` is the location of the `EndSwitch` instruction. `targetOffset` is the location that `EndSwitch` branches to.